ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -189,17 +189,17 @@ public WebElement endVisitLink() {
 	}
 
 	public void visitNote() {
-		clickOnLast(VISIT_NOTE);
+		clickOn(VISIT_NOTE);
```

Comment:
i see on some of the logs you posted,this file breaks because it cant find method clickOnLast